streaming/java/streaming-api/src/main/java/io/ray/streaming/message/KeyRecord.java

```diff
@@ -1,6 +1,8 @@
 package io.ray.streaming.message;
 
 
+import java.util.Objects;
```

Comment:
Shall we take `Retraction` into keyRecord?

Reply:
`Retraction` is not related to this PR, maybe we can discuss it in later PRs.